src/functional/operators.h

```diff
@@ -578,6 +602,10 @@ UNARY(Sqrt,    sqrt,       Ops<ElementType>::sqrt(x));
 UNARY(Neg,     operator-,  Ops<ElementType>::neg(x));
 UNARY(Sgn,     sgn,        Ops<ElementType>::sgn(x));
 
+UNARY(Round,   round,      Ops<ElementType>::round(x));
+UNARY(Floor,   floor,      Ops<ElementType>::floor(x));
+UNARY(Ceil,    ceil,       Ops<ElementType>::ceil(x));
+
 BINARY(Plus,   operator+,  Ops<ElementType>::add(x, y));
 BINARY(Minus,  operator-,  Ops<ElementType>::sub(x, y));
 BINARY(Mult,   operator*,  Ops<ElementType>::mul(x, y));
```

Comment:
quantization requires round and floor. Ceil is added for the sake of completeness